Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18
+FROM node:18-slim
```

Comment:
does it also work with `alpine` tag? alpine is lighter than slim.

Reply:
Oops. Could you please see the comment above? @chonla

Reply:
@kahnwong @zkan 
Thanks for feedback.

Reply:
FYI - https://github.com/thedataengineeringbook/thedataengineeringbook/pull/47